pkgs/by-name/my/myks/package.nix

```
@@ -0,0 +1,49 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, installShellFiles, myks, stdenv
+}:
+
+buildGoModule rec {
+  pname = "myks";
+  version = "3.4.4";
+
+  src = fetchFromGitHub {
+    owner = "mykso";
+    repo = "myks";
+    rev = "v${version}";
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```